modules/core/src/webgl/classes/vertex-array-object.js

```diff
@@ -239,18 +247,19 @@ export default class VertexArrayObject extends Resource {
 
   static _setConstantIntArray(gl, location, array) {
     assert(isWebGL2(gl));
+    // NOTE - array needs to be converted to Int32Array and padded to length 4
```

Comment:
This is interesting. What's the background for this requirement? If there's something in the spec or a bug report that explains it, a link here would be helpful.

Reply:
Oh so those functions just don't exist... Here's the API doc link: https://www.khronos.org/registry/OpenGL-Refpages/es3.0/html/glVertexAttrib.xhtml

Are we using int attributes anywhere?